htdocs/web_portal/views/downtime/added_downtime.php

```diff
@@ -1,5 +1,27 @@
 <div class="rightPageContainer">
     <h1 class="Success">Success</h1><br />
-    New Downtime successfully created. <br />
-    <a href="index.php?Page_Type=Downtime&amp;id=<?php echo $params['dt']->getId() ?>">View new downtime</a>
+    <?php
+    if (count($params['submittedDowntimes']) != 1) {
+        echo '<p>';
+        echo 'New Downtimes successfully created. ';
+        echo 'Please click the links below for more information.';
+        echo '</p>';
+    } else {
+        echo '<p>';
+        echo 'New Downtime successfully created. ';
+        echo 'Please click the link below for more information.';
+        echo '</p>';
+    }
+
+    echo '<ul>';
+    foreach ($params['submittedDowntimes'] as $siteName => $downtimeDetails) {
+        echo '<li>';
+        echo $siteName . ':';
+        echo '<a href="index.php?Page_Type=Downtime&id=';
+        echo $downtimeDetails->getId();
+        echo '"> Downtime ' . $downtimeDetails->getId() . '</a>';
```

Comment:
I would tweak the spacing on this so the space isn't included in the hyperlink text.

```suggestion
        echo $siteName . ': ';
        echo '<a href="index.php?Page_Type=Downtime&id=';
        echo $downtimeDetails->getId();
        echo '">Downtime ' . $downtimeDetails->getId() . '</a>';
```

Reply:
Sure. Should be resolved by 294ebd1283736a0778bf80a6b070150b8a7fab5a.